src/tequila/circuit/compiler.py

```diff
@@ -1297,3 +1315,83 @@ def compile_trotterized_gate(gate, compile_exponential_pauli: bool = False):
         return compile_exponential_pauli_gate(result)
     else:
         return result
+
+
+@compiler
+def compile_ry(gate: RotationGateImpl, controlled_rotation: bool = False) -> QCircuit:
+    """
+    Compile Ry gates into Rx and Rz.
+    Parameters
+    ----------
+    gate:
+        the gate.
+    controlled_rotation:
+        to determine if the gate break down need to be applied here
```

Comment:
comment is a bit hard to understand, maybe:
"determine if the decomposition is performed for controlled gates as well" or something like this

Reply:
done